Specs/Scene/ModelExperimental/ModelExperimentalPrimitiveSpec.js

```diff
@@ -40,6 +54,7 @@ describe("Scene/ModelExperimental/ModelExperimentalPrimitive", function () {
   it("configures the pipeline stages", function () {
```

Comment:
Oh looks like I didn't add any tests for configuring the pipeline with custom shaders. I should do that.